diff -up quickjs-2024-01-13/Makefile.orig quickjs-2024-01-13/Makefile
--- quickjs-2024-01-13/Makefile.orig	2024-10-07 18:37:31.935011886 +0200
+++ quickjs-2024-01-13/Makefile	2024-10-07 18:38:45.730310335 +0200
@@ -122,7 +122,7 @@ endif
 CFLAGS+=$(DEFINES)
 CFLAGS_DEBUG=$(CFLAGS) -O0
 CFLAGS_SMALL=$(CFLAGS) -Os
-CFLAGS_OPT=$(CFLAGS) -O2
+CFLAGS_OPT=$(CFLAGS) -O2 $(RPMPLDCFLAGS)
 CFLAGS_NOLTO:=$(CFLAGS_OPT)
 ifdef CONFIG_COSMO
 LDFLAGS+=-s # better to strip by default
